SDL/Triplet.h

```diff
@@ -377,6 +366,34 @@ namespace SDL {
     // Cut #2: Pointed Z (Inner segment two MD points to outer segment inner MD)
     pass = pass and ((zOut >= zLoPointed) && (zOut <= zHiPointed));
 
+    // First obtaining the raw betaIn and betaOut values without any correction and just purely based on the mini-doublet hit positions
```

Comment:
```suggestion
    // raw betaIn value without any correction, based on the mini-doublet hit positions
```
betaOut does not exist anymore and the note about "First" does not apply anymore, since there is no correction and iteration anymore

Reply:
Applied, thanks!